.github/workflows/release-installer.yaml

```diff
@@ -0,0 +1,105 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
+# "release" runner tag is to make target hosts deterministic for easy clean up.
+# TODO: Remove the "release" runner tag when installer tests are stable.
```

Comment:
`for easy clean up` -> do you mean the cleanup mentioned in https://github.com/runfinch/finch/pull/97/files#r1042749876? If yes, it seems that only arm hosts need that label?

Reply:
This is one reason. The other reason is if the installer candidates of future versions have anything wrong, there is still risk to mess up the host. Even if the host has running job, installer test/release can just wait several minutes as it is normally not time-sensitive. It doesn't hurt to be a bit conservative now. We can remove this when installer release has better automation and self host runners have better cleanup strategy.

Reply:
Thank you. Maybe the followin?

```sh
# "release" runner tag is to make target hosts deterministic for easy clean up.
# This not only refers to the cleanup described in <issue #> for arm hosts.
# Currently tests on amd64 hosts sometimes also need manual cleanup via SSHing to the host.
# TODO: Remove the "release" runner tag when installer tests are stable.
```